hugo/content/1-basics/installation.md

```diff
@@ -0,0 +1,43 @@
++++
+next = "/1-basics/quick-tour"
+prev = "/1-basics/what-is-mangrove"
+title = "Installation"
+toc = true
+weight = 2
+
++++
+
+To get Mangrove up and running in your project, simply follow these instructions to install Mangrove on your system.
+
+## Prerequisites
+
+1. OS X or any other standard 'nix platform.
+2. A modern compiler that supports C++14.
+	* Mangrove has been confirmed to work with Clang 3.8+, Apple Clang 7.3+, or GCC 6.1+.
+3. CMake 3.2+. 
+4. The MongoDB C driver version 1.3.5+. (see below)
+5. The MongoDB C++ driver version 3.0.2+. (see below)
```

Comment:
Depending on when we release these docs, it's worth keeping in mind that libmongocxx v3.0.2 isn't released yet.